### PR TITLE
[Item Mods] Rename DA_DOUBLE_DAMAGE and TA_TRIPLE_DAMAGE mods

### PR DESCRIPTION
**Pull request by Schaedelschaden**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Fulfills TeoTwawki's request to rename mods in #2088

- Update DA_DOUBLE_DAMAGE and TA_TRIPLE_DAMAGE mods to DA_DOUBLE_DMG_RATE and TA_TRIPLE_DMG_RATE to reduce future confusion on the mods.
- Update attackutils.cpp with the renamed mods.
- Update gear_sets.lua with the renamed mods.
- Update the documentation/modifier.h ordered by ID.txt with the most recent mods.

### CODE REVIEW

**Comment by TeoTwawki** [comment]
https://git-scm.com/book/en/v2/Git-Branching-Rebasing

this is a skill you absolutely need

https://www.youtube.com/watch?v=_UZEXUrj-Ds

**Comment by Schaedelschaden** [comment]
Absolutely, been missing that! Sorry for the weird spamming. Want me to redo these PR's?

**Comment by TeoTwawki** [comment]
> Absolutely, been missing that! Sorry for the weird spamming. Want me to redo these PR's?

I'm a turtle person m'self https://www.youtube.com/watch?v=qrMubRPkkrE

you don't have to close them. just rebase on local copy then force push, it will automatically revise the pr

better vid: https://www.youtube.com/watch?v=ElRzTuYln0M

**Comment by TeoTwawki** [comment]
how this went astray here is you wound up with commits from your other pr's in here then tried to remove those changes, but it doesn't undo tho proper - its same commit in both branches, so say we merge those in, then merge a pr where those are undone. the end result is the changes in the 1st pr go missing from the project because the 2nd pr removed the changes, not the commits.

Always use a clean 1:1 copy of our base branch as your starting point. Avoid use of the main or "master" branch of your fork for any prs so that it stays clean for starting new branches (by default, that will be the basis of any new branch)

**Comment by Schaedelschaden** [comment]
Thank you for the heads up! I knew I needed to have fresh branches for changes but hadn't set up the process properly yet. I appreciate the information for getting it set up properly and will try to get these two PR's straightened out.

File: scripts/globals/weaponskills.lua [hunk 1]
```diff
@@ -189,8 +189,10 @@ local function getMultiAttacks(attacker, target, numHits)
             bonusHits = bonusHits + 3
         elseif math.random() < tripleRate then
             bonusHits = bonusHits + 2
+            attacker:setLocalVar("TripleAttackDMG", 2)
```
Comment (TeoTwawki) [inline scripts/globals/weaponskills.lua]: I don't think we want to call that function for variable handling here. This should be doable without setting them on the entity.

https://github.com/LandSandBoat/server/wiki/Understanding-variables-%E2%80%94-a-brief-guide

Comment (Schaedelschaden) [reply scripts/globals/weaponskills.lua]: That's from #2080 before I removed it due to the mods not affecting weaponskills. I'll keep in mind that I should try not to use localvars though, thank you!

**Comment by Schaedelschaden** [comment]
Ok, I pulled the current LSB base, created the modUpdates branch, changed the files, and force pushed the changes to the commit. Could I have someone take a look to see if I managed to get it straightened out or if I still need to do other things please?

**Comment by WinterSolstice8** [comment]
The CI says the build is failing, check out the logs

File: scripts/globals/status.lua [hunk 2]
```diff
@@ -1339,8 +1339,8 @@ xi.mod =
     MAGIC_DAMAGE                    = 311, --  Magic damage added directly to the spell's base damage
 
     -- Gear set modifiers
-    DA_DOUBLE_DAMAGE                = 408,  -- Double attack's double damage chance %.
-    TA_TRIPLE_DAMAGE                = 409,  -- Triple attack's triple damage chance %.
+    DA_DOUBLE_DMG_RATE              = 408,  -- Double attack's double damage chance %.
```
Comment (TeoTwawki) [inline scripts/globals/status.lua:1342]: missed this in modifier.h 

also I see other edits in modifier.h that i am confused by: every time someone adds a mod, it should be both there and in status.lua - were these other mods _MISSING?_

Comment (Schaedelschaden) [reply scripts/globals/status.lua:1342]: To my knowledge the mods were not outright missing from either status.lua or modifier.h and only had to have their names updated. The CHAKRA_MULT and CHAKRA_REMOVAL mods in modifier.h I only removed a space before the comment to align them with the other mods nearby.

Comment (TeoTwawki) [reply scripts/globals/status.lua:1342]: I mean these:
```cpp
    FULL_CIRCLE                  = 1025, // Increases the initial multiplier on MP returned via Full Circle
    CHAKRA_MULT                  = 1026, // Chakra multiplier increase (from gear)
    CHAKRA_REMOVAL               = 1027, // Extra statuses removed by Chakra
    BOLSTER_EFFECT               = 1028, // Adds bonus duration as +N seconds
    LIFE_CYCLE_EFFECT            = 1029, // Adds bonus HP% returned to the luopan when using Life Cycle
    AURA_SIZE                    = 1030, // Used to extend aura size, the formula is 6.25 + (PEntity->getMod(Mod::AURA_SIZE) / 100) so adding 100 will make this 7.25
    AUGMENT_CONSERVE_MP          = 1031, // Percent chance to deal extra damage based on Conserve MP Amount (BLM AF3 Sets)
    AUGMENT_COMPOSURE            = 1032, // Percent Enhancing Duration Extension for Others (RDM AF3 Sets)
    AUGMENT_DAMAGE_HP            = 1033, // Percent chance to increase damage based on player HP% (DRK AF3 Sets)
    AUGMENT_DAMAGE_PET_HP        = 1034, // Percent chance to increase damage based on pet HP% (BST/DRG AF3 Sets)
    AUGMENT_BLOOD_BOON           = 1035, // Percent chance to deal extra damage based on Blood Boon Amount (SMN AF3 Sets)
    AUGMENT_BLU_MAGIC            = 1036, // Percent chance for BLU magic to receive 3x WSC value for spell (BLU AF3 Sets)
    GEOMANCY_MP_NO_DEPLETE       = 1037, // Percent chance for Geomancy to cost 0 MP (GEO AF3 Sets)
```

Comment (TeoTwawki) [reply scripts/globals/status.lua:1342]: oh! doc file

Comment (TeoTwawki) [reply scripts/globals/status.lua:1342]: I was so confused by seeing "modifier.h" in the documents NAME

Comment (Schaedelschaden) [reply scripts/globals/status.lua:1342]: Those were missing from documentation/modifier.h, but they existed in src/map/modifier.h and scripts/globals/status.lua.

**Comment by Schaedelschaden** [comment]
Is there anything I can do to fix the bind error in the Full_Startup_Checks?

**Review by TeoTwawki — APPROVED** [review]
passing ci and looks ok now

**Pull request merged**